[OT294-75]-Endpoint Creacion Miembros

LuciaCerpa opened the pull request:
https://alkemy-labs.atlassian.net/jira/software/c/projects/OT294/boards/388?modal=detail&selectedIssue=OT294-75
![Validando-nombre](https://user-images.githubusercontent.com/102775738/193125256-d8fc7c5f-726b-4420-8723-a88f53eba3a5.png)
![MiembroCreado](https://user-images.githubusercontent.com/102775738/193125116-8355a439-6510-4fa1-9e7c-dcc3be3aac1a.png)
![MiembroCreado2](https://user-images.githubusercontent.com/102775738/193125132-43d2eb22-7d58-4ded-a402-98a9e187aafb.png)

lucy1528 committed:
Endpoint Creacion Miembros

FerLeiva commented on validators/members.validator.js:
```diff
@@ -0,0 +1,13 @@
+const createMemberValidators = [
+    body("name")
+        .notEmpty()
+        .withMessage("Nombre de la categoria no puede ser un campo vacio")
```
Error de tipeo en el mensaje, dice nombre de la categoria

LuciaCerpa replied:
Cambios realizados

LuciaCerpa replied:
![cambioValidatorMembers](https://user-images.githubusercontent.com/102775738/193323178-a052130b-58a0-46bd-8fb3-5fc1033465da.png)

FerLeiva reviewed (changes requested):
Error de tipeo

LuciaCerpa committed:
Update members.validator.js

Corrección de sintaxis en el mensaje de respuesta del validador de Nombre en miembro

LuciaCerpa committed:
Merge branch 'main' into OT294-75
